lib/App/Asciio/Actions/Unsorted.pm

```diff
@@ -91,6 +92,8 @@ my ($self, $direction) = @_ ;
 
 my ($family, $size) = $self->get_font() ;
 
+my ($ori_character_width, $ori_character_height) = $self->get_character_size() ;
```

Comment:
This is usually a bad move.

- what does ori mean? yes it took me a minute to figure it out because it could be something new that's called ori.


in the worst case, old and new are are short and carry the right meaning

but don't use old if you can, it's better to let variable named normally and new variable use "new", it's a good habit even if I do the error now then myself 

my ($character_width, $character_height) = $self->get_character_size() ;  # better